__tests__/app.test.js

```diff
@@ -160,6 +160,22 @@ describe("news-api", () => {
                     expect(msg).toBe("bad request")
                 })
             })
+        })
+        describe("GET /api/user", () => {
+            test("responds with status code 200 and an array of user objects in the expected format", () => {
+                return request(app)
+                .get("/api/user")
+                .expect(200)
```

Comment:
This end point name doesnt seem accurate, should this return one user or many users?